frontend/src/components/CreateNearAccountModal.tsx

```diff
@@ -120,21 +113,16 @@ export const CreateNearAccountModal = ({
     }
   };
 
-  // Use Dialog component from shadcn/ui
   return (
-    <Dialog open={isOpen} onOpenChange={(open) => !open && handleClose()}>
-      <DialogContent
-        className="sm:max-w-[425px]"
-        onEscapeKeyDown={handleClose}
-        onPointerDownOutside={handleClose}
-      >
-        <DialogHeader>
-          <DialogTitle>Choose Your NEAR Account Name</DialogTitle>
-          <DialogDescription>
+    <Modal isOpen={isOpen} onClose={onClose}>
+      <div className="max-w-[425px]">
```

Comment:
I don't think this max-w-425 is doing anything I think the size of content is dictating the width rn